packages/@react-spectrum/slider/src/RangeSlider.tsx

```diff
@@ -60,7 +60,7 @@ function RangeSlider(props: SpectrumRangeSliderProps, ref: FocusableRef<HTMLDivE
         let upperTrack = (
           <div
             className={classNames(styles, 'spectrum-Slider-track')}
-            style={{[cssDirection]: `${state.getThumbPercent(1) * 100}%`, width: `${(1 - state.getThumbPercent(1)) * 100}%`}} />
```

Comment:
What was this change for?

Reply:
Ah this one I can answer: @ktabors had noticed the following issue with range sliders where the right most track line spills into the right thumb:
![image](https://user-images.githubusercontent.com/9661127/102405165-90ff5500-3f9d-11eb-982a-751926c3398c.png)
Looking at https://opensource.adobe.com/spectrum-css/slider.html#range, the right most track line only has width applied to it, the `left`/`right` inline style here was extraneous